scripts/globals/abilities/elemental_sforzo.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------
+-- Ability: Elemental Sforzo
+-- Grants immunity to all magic attacks.
+-- Obtained: Rune Fencer Level 1
+-- Recast Time: 1:00:00
+-- Duration: 00:00:30
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+
+function onAbilityCheck(player,target,ability)
+  return 0,0
+end
+
+function onUseAbility(player,target,ability)
+  -- TODO: Add enmity when cast.
```

Comment:
there are fields in the db table for the CE and VE, if the amounts are known

Reply:
Cumulative Enmity and Volatile Enmity. One of them decays over time, they other does not.

Reply:
Looks like those values aren't populated: https://github.com/project-topaz/topaz/blob/release/sql/abilities.sql#L293

It's listed as `1800` CE and `7200` VE here: 
https://www.bg-wiki.com/bg/Elemental_Sforzo

Reply:
That's why I was telling you where to populate them.

Reply:
Done. Please take another look.